src/drt/doc/Distributed.md

```diff
@@ -0,0 +1,139 @@
+
+
+
+# Documentation for distributed detailed routing with Kubernetes and Google cloud.
+
+In this tutorial, we introduce how to use distributed detailed routing with a Kubernetes cluster. We will be using a cluster running in google cloud, but the process is similar to using any Kubernetes cluster.
+
+
+## General System Design
+
+
+
+
+![alt_text](gsd.jpg "System Components")
+
+
+The Diagram above shows four main components to run distributed routing:
+
+* The leader: responsible for orchestrating the detailed routing process through creating the routing jobs that the workers need to handle and aggregating the routed chunks.
+* The load balancer: the leader sends the needed jobs to the load balancer which is responsible for balancing the jobs fairly among the workers and handle failed ones.
```

Comment:
In what way does the load balancer interact with NFS?

Reply:
It used to need for the tcl script it runs on but that can be removed now.